README.md

```diff
@@ -194,7 +195,7 @@ Running Tests
 -------------
 
 The tests are run using the supplied `Makefile`.
-First, run `make split-tests` to generate some of the tests from the markdown files.
+First, run `make venv && make build-prove` to generate some of the tests from the markdown files.
```

Comment:
I think `venv` is a (transitive) prerequisite of `build-prove`. Can you confirm?

Reply:
Actually I just noticed @ehildenb added instructions to run `make venv` after `make build`, so this is no longer relevant. I'm crossing it out.